src/olympia/lib/settings_base.py

```diff
@@ -683,20 +681,34 @@ def JINJA_CONFIG():
             'js/common/keys.js',
 
             # jQuery UI
-            'js/lib/jquery-ui/core.js',
-            'js/lib/jquery-ui/position.js',
-            'js/lib/jquery-ui/widget.js',
-            'js/lib/jquery-ui/menu.js',
-            'js/lib/jquery-ui/mouse.js',
-            'js/lib/jquery-ui/autocomplete.js',
-            'js/lib/jquery-ui/datepicker.js',
-            'js/lib/jquery-ui/sortable.js',
+            'js/node_lib/ui/version.js',
+            'js/node_lib/ui/data.js',
+            'js/node_lib/ui/disable-selection.js',
+            'js/node_lib/ui/ie.js',
+            'js/node_lib/ui/keycode.js',
+            'js/node_lib/ui/escape-selector.js',
+            'js/node_lib/ui/labels.js',
+            'js/node_lib/ui/jquery-1-7.js',
```

Comment:
Is this something only related to ui then? Just wanted to make sure this isn't an actual jquery lib because presumably we're already bringing that in elsewhere?